src/topology/subset_properties.lean

```diff
@@ -200,6 +207,8 @@ section tube_lemma
 
 variables [topological_space β]
 
+/-- `nhds_contain_boxes s t` means that any neighborhood of `s × t` in `α × β` includes
```

Comment:
This is not what the lemma says. It is not talking about neighborhoods, but about open sets (neighborhoods need not be open). The name is wrong, in fact. If you want to fix the name, great, but at least you should fix the docstring :)

Reply:
Added `open neighborhood`. I don't want to fix the name right now.